eng/Version.Details.xml

```diff
@@ -31,37 +31,25 @@
       <Uri>https://github.com/dotnet/coreclr</Uri>
       <Sha>fd22fd600a2526a7c58536da0e80325eb710d7f1</Sha>
       <RepoName>coreclr</RepoName>
-    </Dependency>
-    <Dependency Name="Microsoft.NETCore.Runtime.CoreCLR" Version="3.1.2-servicing.20066.2" CoherentParentDependency="Microsoft.Private.CoreFx.NETCoreApp">
-      <Uri>https://github.com/dotnet/coreclr</Uri>
-      <Sha>fd22fd600a2526a7c58536da0e80325eb710d7f1</Sha>
       <RepoName>coreclr-portable</RepoName>
     </Dependency>
-    <Dependency Name="NETStandard.Library" Version="2.1.0" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+    <Dependency Name="NETStandard.Library" Version="2.1.0" CoherentParentDependency="Microsoft.NETCore.App.Internal">
       <Uri>https://github.com/dotnet/standard</Uri>
       <Sha>a5b5f2e1e369972c8ff1e2183979fab6099f52ef</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Private.CoreFx.NETCoreApp" Version="4.7.0-servicing.20067.2" CoherentParentDependency="Microsoft.NETCore.App">
+    <Dependency Name="Microsoft.Private.CoreFx.NETCoreApp" Version="4.7.0-servicing.20067.2" CoherentParentDependency="Microsoft.NETCore.App.Internal">
       <Uri>https://github.com/dotnet/corefx</Uri>
       <Sha>e946cebe43a510e8c6476bbc8185d1445df33a1a</Sha>
       <RepoName>corefx</RepoName>
-    </Dependency>
-    <Dependency Name="Microsoft.Private.CoreFx.NETCoreApp" Version="4.7.0-servicing.20067.2" CoherentParentDependency="Microsoft.NETCore.App">
-      <Uri>https://github.com/dotnet/corefx</Uri>
-      <Sha>e946cebe43a510e8c6476bbc8185d1445df33a1a</Sha>
       <RepoName>corefx-portable</RepoName>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App" Version="3.1.2-servicing.20067.4" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+    <Dependency Name="Microsoft.NETCore.App.Internal" Version="3.1.2-servicing.20067.4" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
```

Comment:
I would recommend `Microsoft.NETCore.App.Runtime.linux-x64` instead. The internal package isn't *likely* to be removed in 3.1, but it shouldn't be used by anyone directly. Using the runtime pack is also in line with `Microsoft.AspNetCore.App.Runtime.linux-x64` below (the scenario is the same).